tests/suite/dataplane_perf_test.go

```diff
@@ -56,50 +56,55 @@ var _ = Describe("Dataplane performance", Ordered, Label("performance"), func()
 	}
 
 	BeforeAll(func() {
-		pwd, err := os.Getwd()
-		Expect(err).ToNot(HaveOccurred())
-		parentDir = filepath.Dir(pwd)
-		resultsFilePath = fmt.Sprintf("%v/results/dp-perf/%v/", parentDir, version)
-		Expect(os.MkdirAll(resultsFilePath, 0o777)).To(Succeed())
-		resultsFilePath = fmt.Sprintf("%v/%v.md", resultsFilePath, version)
-		if portFwdPort != 0 {
-			port = fmt.Sprintf(":%s", strconv.Itoa(portFwdPort))
-		}
-		addr = fmt.Sprintf("http://%s%s", address, port)
 		Expect(resourceManager.Apply([]client.Object{ns})).To(Succeed())
 		Expect(resourceManager.ApplyFromFiles(files, ns.Name)).To(Succeed())
 		Expect(resourceManager.WaitForAppsToBeReady(ns.Name)).To(Succeed())
-		outFile, err = os.OpenFile(resultsFilePath, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0o777)
-		Expect(err).To(BeNil())
+
+		port := ":80"
+		if portFwdPort != 0 {
+			port = fmt.Sprintf(":%s", strconv.Itoa(portFwdPort))
+		}
+		addr = fmt.Sprintf("%s%s", address, port)
+
+		resultsDir, err := framework.CreateResultsDir("dp-perf", version)
```

Comment:
Can we simplify things and create the results directory when we create the file?

Reply:
I have these as separate calls because the upgrade test needs the results dir for other files.